src/main/webapp/app/account/activate/activate.component.ts

```diff
@@ -1,25 +1,26 @@
-import { Component, OnInit } from '@angular/core';
-import { ActivatedRoute } from '@angular/router';
-
+import { Component, OnInit, inject } from '@angular/core';
+import { ActivatedRoute, RouterLink } from '@angular/router';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
 import { ActivateService } from './activate.service';
 import { ProfileService } from 'app/shared/layouts/profiles/profile.service';
 import { mergeMap } from 'rxjs/operators';
+import { TranslateDirective } from 'app/shared/language/translate.directive';
 
 @Component({
     selector: 'jhi-activate',
     templateUrl: './activate.component.html',
+    standalone: true,
+    imports: [TranslateDirective, RouterLink, ArtemisSharedModule],
```

Comment:
Consideration for the future: If everything is migrated to Standalone maybe we can make the ArtemisSharedModule smaller and just import only the things we need directly in the component.

Reply:
good point for the future, but for now this is out of scope